### PR TITLE
Avoid bundling ol imports into the final build

### PR DESCRIPTION
**Pull request by menghany**

### CODE REVIEW

**Comment by jonataswalker** [comment]
This is great but the `iife` bundle have some problems.

You'll see if you run `examples/contextmenu.html`. Making some tests it looks like that this change can fix it:

```javascript
rollupOptions: {
    external: [/^ol.*/],
    output: {
        globals: {
            'ol/MapBrowserEvent': 'ol.MapBrowserEvent',
            'ol/control/Control': 'ol.control.Control',
        },
        assetFileNames: () => 'ol-contextmenu.css',
    },
},
```

File: vite.config.ts [hunk 1]
```diff
@@ -39,7 +39,12 @@ export default defineConfig(({ command }) =>
                       formats: ['es', 'umd', 'iife'],
                   },
                   rollupOptions: {
+                      external: ['vue', /^ol.*/],
```
Comment (jonataswalker) [inline vite.config.ts:42]: `'vue'` here is not necessary, right?

File: vite.config.ts [hunk 2]
```diff
@@ -39,7 +39,12 @@ export default defineConfig(({ command }) =>
...
                       output: {
+                          globals: {
+                              'ol/MapBrowserEvent': 'MapBrowserEvent',
```
Comment (jonataswalker) [inline vite.config.ts:45]: `'ol/MapBrowserEvent': 'ol.MapBrowserEvent'`

not tested with examples that import `ol-contextmenu`, could you do that, please?

File: vite.config.ts [hunk 3]
```diff
@@ -39,7 +39,12 @@ export default defineConfig(({ command }) =>
...
+                              'ol/control/Control': 'Control',
```
Comment (jonataswalker) [inline vite.config.ts:46]: `'ol/control/Control': 'ol.control.Control'`

**Review by jonataswalker — CHANGES_REQUESTED** [review]

File: vite.config.ts (hunk 1 above)
Comment (menghany) [reply vite.config.ts:42]: Sorry, I forgot to delete my test code.

Comment (menghany) [reply vite.config.ts:42]: Sorry, I forgot to delete my test code.

**Review by jonataswalker — APPROVED** [review]

**Pull request merged**